src/newmagic.cpp

```diff
@@ -1700,6 +1708,205 @@ void cast_manipulation_spell(struct char_data *ch, int spell, int force, char *a
+      // NOTE: Added sidestep here. Not sure if you should be able to sidestep lightning, but if you can dodge it in the first place...
+      success -= success_test(GET_DEFENSE(vict) + GET_DEFENSE(vict) ? GET_POWER(vict, ADEPT_SIDESTEP) : 0, 4 + damage_modifier(vict, buf, sizeof(buf)));
+    }
+    act("A thin laser beam is emitted from $n and heads directly towards $N!", TRUE, ch, 0, vict, TO_ROOM);
+    if (!IS_NPC(ch) && PLR_FLAGGED(ch, PLR_KILLER) && !IS_NPC(vict))
+      success = -1;
+    else success += success_test(skill, target + 4);
+    if (success > 0 && GET_REFLECT(vict) && (reflected = reflect_spell(ch, vict, spell, force, 0, target + 4, success))) {
+      vict = ch;
+      ch = temp;      
+    }
+    if (success <= 0) {
+      act("$n easily dodges it, and the beam flickers and vanishes.", FALSE, vict, 0, 0, TO_ROOM);
+      send_to_char("You easily dodge it!\r\n", vict);
+    } else {
+      success -= success_test(GET_BOD(vict) + GET_BODY(vict) + GET_POWER(ch, ADEPT_TEMPERATURE_TOLERANCE), force - (GET_IMPACT(vict) / 2));
+      int dam = convert_damage(stage(success, basedamage));
+      if (!AWAKE(vict)) {
+        act("$n's body recoils in pain as the laser beam finds its mark.", TRUE, vict, 0, 0, TO_ROOM);
+        send_to_char("You feel an uncomfortably painful burning sensation where the beam struck.\r\n", vict);
+      } else if (GET_MENTAL(vict) - (dam * 100) <= 0) {
+        act("$n is cleanly pierced by the laser beam, the hole cut in $s body smoking as they drop like a rock.", TRUE, vict, 0, 0, TO_ROOM);
+        send_to_char("You feel your body cleanly pierced by the laser beam, which leaves a smoking hole in your body.\r\n", vict);
+      } else if (GET_MENTAL(vict) - (dam * 100) <= 300) {
+        act("$n is almost pierced by the laser beam, pain filling $s body as a thin wisp of smoke rises from $m.", TRUE, vict, 0, 0, TO_ROOM);
+        send_to_char("Your body is filled with intense pain as the laser beam strikes you, inflicting a cauterized wound.\r\n", vict);
+      } else if (GET_MENTAL(vict) - (dam * 100) <= 700) {
+        act("$n jolts back as the laser strikes $m, $s body shuddering in pain as the laser beam dissipates.", TRUE, vict, 0, 0, TO_ROOM);
+        send_to_char("Pain flashes through your body as the laser strikes, your body shuddering at the intense heat.\r\n", vict);
+      } else if (dam > 0) {
+        act("$n visibly flinches as the laser connects, but otherwise seems fine.", TRUE, vict, 0, 0, TO_ROOM);
+        send_to_char("You flinch as the laser strikes you, feeling an intense heat for a moment.\r\n", vict);
+      } else {
+        act("The laser beam strikes $n, but seems to be easily absorbed.", FALSE, vict, 0, ch, TO_ROOM);
+        send_to_char("Your body absorbs the laser beam without injury.\r\n", vict);
+      }
+      for (int i = 0; i < NUM_WEARS; i++)
+        if (GET_EQ(vict, i) && (GET_OBJ_MATERIAL(GET_EQ(vict, i)) == 10 || GET_OBJ_MATERIAL(GET_EQ(vict, i)) == 11))
```

Comment:
This is an artifact of the lightning spell code. Two things:

1) Materials 10 and 11 are electronics and computers respectively, so if you're going to keep this, please change this line to read `if (GET_EQ(vict, i) && (GET_OBJ_MATERIAL(GET_EQ(vict, i)) == MATERIAL_ELECTRONICS || GET_OBJ_MATERIAL(GET_EQ(vict, i)) == MATERIAL_COMPUTERS))`

2) I would swap this out for something with a random chance to it (`if (!number(0, 50))` maybe?) or just remove the for-loop altogether.

Reply:
Went with option 2, because random gear damage is fun.